app/styles/component/_recirculation.scss

```diff
@@ -30,17 +30,28 @@
 			background-color: #f08a24;
+			left: 0;
+		}
+
+		&-plista {
+			background: #666 url('data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAIoAAAAeCAMAAADErnYDAAAAAXNSR0IB2cksfwAAAppQTFRFAAAAp6enqqqqqqqqqqqqrKysqqqqqqqqqqqqqqqqpKSkqqqqqqqqqqqqq6ur////gICAqampq6urqampqqqqq6urqampqqqqqqqqqqqqqampq6urq6urqqqqqqqqqqqqqqqqn5+fqqqqp6enqqqqr6+vqqqqqqqqpqamqqqqqqqqqamppaWlqqqqqqqqqqqqqqqqrq6uqKioqqqqq6urmZmZrKysqqqqqqqqqqqqv7+/qqqqqqqqqqqqqqqqqqqqqqqqqampqampqqqqqampq6urq6urqqqqqqqqtra2qqqqqampqampqqqqs7Ozqqqqqqqqqqqqqqqqq6urqqqqqqqqqqqqq6urq6urqKiorKysq6urqqqqqqqqq6urq6urqqqqqqqqqqqqqqqqra2tq6urqqqqqqqqqqqqq6urqqqqqqqqqampqqqqqqqqqqqqqqqqqampqqqqqampra2tqqqqq6urrKysqqqqqqqqqqqqqKioqqqqqqqqqampqqqqqqqqqqqqp6enqqqqqampqqqqq6urqqqqqqqqqqqqqampqqqqqqqqqqqqq6urqampqqqqqqqqqampqqqqqqqqq6urqqqqqqqqqqqqqqqqqqqqqampqqqqsbGxq6urqamprq6uqqqqqqqqrKysqqqqq6urqqqqrKysra2tqampqampqqqqqqqqqqqqq6urqqqqqKioqqqqqqqqq6urq6uroqKirKysqqqqqampqqqqra2tqqqqqqqqqqqqqqqqq6urqqqqqampqKioqqqqqampqampq6urq6urqqqqqampqqqqqqqqq6urqampqqqqq6urqqqqq6urqqqqqqqqqampqqqqqKioqqqqqqqqq6urqqqqqqqqqqqqqampitLcJAAAAN50Uk5TABrt/tgrA8z/xA76+/2UAQJicJLsOnH8G2B0Rqbb9/LHCLggCRDI0hfU9YARr9O10RM176kFN5Pw5QTDuycYs8treplHoFLFBgfXjF+oCvHuRd95Y9o8TIIyLo7k+HZq9vTJZhmR3Zwebc3BUDOw4vmkKn0f45clq96WJuvgnubowB22lX6dh64hVkgPOYhTvwxEwhVzuhJsvWmYdQ1AhhaBMCiyVdYxHFlKeEvzXrQj3M+jPQs0JIMtIlTQP+dkQokpim47YWetocrhi4+xmm9bucabzi+s6U+Qe06ntlSl2gAABJxJREFUeJztlvtbFFUYx18ZHL7BrLuGMUm6XGSJRYTSkEskKCC7gkIUYbLIUsll8Z4rqBioeCkS8gJGJFEiYYagZHQhKzWjq93v/S+dc2YX2AWfHp6n2p6n3h9m3jPznjOfeW/nEP2XZYaf5D+TZF9jcAkAk9sC/w0wQYxE0WEW6Q2+RlG4VxAkSbPJoPc5yu0UDJ2COXdMEaUQqHdOHM9FqKbcNW++MSw8JILrkVgw5doLEDktFB2iIsgUDVXF3TGTYMyxWDgFStwiSPEJ4Ubcc+9fiKKCfV+mxUt4ytyXqPeAWYqk5JTUSSjy/Uh7gN2XpSPDcGsUeVnc9FF4nixnUdJhBRkm5G8msrKxchJKKHIsYmhdxd/eCmWaoqHoKTeP5NU8ZYwrx/N3TX5B4oNIE7ocYjYWPvSwQCnCIy6L4rWPcpR1JbZS//XCoXJZgL38scf5Eq4APbGhwlhYWcXV1GqHPaxmo5i6afOWrbZt2yd55Ulgh4Vm+POUcY61vDzsJDlFquV6HXbtrtvq5Ch7UO9RbJHYmzxvdTlymZ6YAGflUw2oM7hR9jRiX+5mKWU/IzmAiqZgJw6yxzPzkw8dPqLmP+2N8gxg5SnTbJR0KnKe1YJUg6NELeITrXjuGNFRO0c5jhPkiRJ/kmidkswWakN7INGpcDzvRunAC+y6HJ1ELyKdldxph7SUaCe6iJu/5I1C3VF8xPrcy4Ck4hU+qsV8dj0DPza/B5v4oxKOchYJXihn+S0DMUS90g6uvwqzC6UPASI0LeeIKtDH9ddQydPwPHNtVNV4ZmooBlNBEwmny/Q6a3m6fq4fFp6kaFwgspWKORs5ShfCvVBExAcwSFG4qD3Lh1VDKUaw2+4UYhdyKcEllipGNA5diJmwjMsrGyToUzWULIaiFnA1BS5hX7b7CfM3OMp++LlL/vKb8lgFDWCYYuDQXvQiTkPJw1vubw2618NeNnp7aAn7554+b5RZADEUWS9nMhAo7xD/n4udQuwsXk5JNNZ3RQWN4D3X9Lmo8UBZg0btxRVFr6G0urxymSUBijy8KS8+b57gYBfKbo7CingFJEViWwGPxiK8r9lks5Q7gg+4+qFAKUO01veOmVHmgUKrcIbrV3kMBMowMvgD/RWbTCmSKOnaa61E15N4QkTUwzIZxWChvCDecg8ZxC7dHRQbqNl8hP7TzQgwEW0v17qtA2mD7D7cgRGLJ0obOlilmfah2l1BObjBrh8jm2gt2tlLSxovr1GwPKYqo20qrzSHgY1yTK6uEoptbiP+70lwNn1SXqTtQd2fQro0MKLAzLqFBwprHZ91ZtowOtZXrtaj/fN07GK7lfUEtmR+UYQDFt5XdF+23PSTzk2BUsHbm238RNeL426jrxDP0sJhb7jW7NqZLV9/U1ga9u11K3mhiIaan3NjQret+s6mNNwUtWJp+96e7KgWs3748Sd1zkDXeOqMp61OkUp/9uWhheWGiaH8AlXCrx5b4T+PAoxmRf5m7cfvqXrfHnAZigTFyP3h65M2P9uq6CE59c9t/2ap4F5Z72sKIcUIGrLofXzW/1+mL38A8Q75z/jUvY4AAAAASUVORK5CYII=');
```

Comment:
Can we use a separate file? This way, we will load it for **all users** and it will postpone the first render.